features/old/accounts/invoices.feature

```diff
@@ -36,3 +36,18 @@ Feature: Show invoices from account's page (#16015909)
     And I follow "2011-01-00000001"
     # TODO: Properly check if order is opened
     Then I should see "Add"
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And the provider is charging its buyers in prepaid mode
+    And an application plan "Fixed" of provider "foo.3scale.localhost" for 0 monthly
+    And a buyer "zoidberg" signed up to application plan "Fixed"
+    And an invoice of buyer "zoidberg" for January, 2011
+    And I go to the invoices issued by me
+    Then I should see the list of years with invoices
```

Comment:
This PR specifically fixes the issue where the provider is seeing the years of invoices that this specific provider doesn't have.
So, I think this is what is needed here - add some invoices for another provider, and make sure that the current provider doesn't see the years of that another one.

Reply:
Added